utils/workflow.go

```diff
@@ -34,6 +34,7 @@ const defaultImageTag = ":latest"
 // GetJSONWorkflow return a Kogito compliant JSON format workflow as bytearray give a specific workflow CR
 func GetJSONWorkflow(workflowCR *operatorapi.KogitoServerlessWorkflow, ctx context.Context) ([]byte, error) {
 	logger := ctrllog.FromContext(ctx)
+	// TODO make sure it still needed.
```

Comment:
Can we have a JIRA to track it?

Reply:
Actually, we can reuse this to set the metadata only, I've updated it.